Makefile

```diff
@@ -73,10 +73,8 @@ vendors:
 	npm run vendors
 installjs:
 	npm install
-testjsfx:
```

Comment:
Not sure why you removed this ?

Reply:
This is not cross-platform

Reply:
It's because, with the ES modules you need a browser to run. Instead of using firefox directly, my proposal here is to print an URL, and then you can open it the way you want.

Would that work for you?

Reply:
Hehe, not a big deal :)

I remember seeing that `open path/to/index.html` would be cross platform, no ? 

At least, I do use `make testjsfx`, so I'd prefer if it still works :)

Reply:
As far as I understand, it has been merged with `testjs` so it still works somehow but maybe the `mocha-phantomjs` call was still useful?

Reply:
Ah, if `open` works that would be perfect, but I think it's not cross-platform. Does it work on your OS?

Reply:
In mine, yes :)

Reply:
> maybe the mocha-phantomjs call was still useful?

It wasn't working for me. If it's useful for somebody I can put it back of course :-)

Reply:
No, that one is broken, it can be removed for sure

Reply:
See #1472